### PR TITLE
Portfolio Mobile version

### PR DESCRIPTION
**Pull request by Josphat205**
This is what I added to my project

- Added the About section which contains my story section and projects
- Added contact section which has an input for the message, email, and text
- Added My social links Twitter, LinkedIn, GitHub, and Pinterest
- Added all  fonts required, Roboto inter and Oswald

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -255,3 +265,255 @@ ul li {
   display: flex;
   justify-content: flex-end;
 }
+
+/* ----------main content section start-- */
+.main-content {
+  display: flex;
```
Comment (Lucas-Masaba) [inline style.css:271]: - [ ] In this project, you are required to wrap the main container with `display: grid`. It is a core requirement and it should be done 👍

File: index.html [hunk 2]
```diff
@@ -5,86 +5,206 @@
...
+                 </div>
+                 <div class="tags">
+                    <h5>css</h5>
+                    <img src="img/Rectangle.png" alt="">
+                    <h5>html</h5>
+                    <img src="img/Rectangle.png" alt="">
+                    <h5>bootstrap</h5>
+                    <img src="img/Rectangle.png" alt="">
+                    <h5>Ruby</h5>
+                  </div>
```
Comment (Lucas-Masaba) [inline index.html:134]: - [ ] The list of technologies are all supposed to be wrapped in a `ul` tag with each individual technology having a `li` tag 👍

File: index.html [hunk 3]
```diff
@@ -5,86 +5,206 @@
...
+              </div>
+              <div class="tags">
+                <h5>css</h5>
+                <img src="img/Rectangle.png" alt="">
+                <h5>html</h5>
+                <img src="img/Rectangle.png" alt="">
+                <h5>bootstrap</h5>
+                <img src="img/Rectangle.png" alt="">
+                <h5>Ruby</h5>
+              </div>
```
Comment (Lucas-Masaba) [inline index.html:113]: - [ ] The list of technologies are all supposed to be wrapped in a `ul` tag with each individual technology having a `li` tag 👍

File: index.html [hunk 4]
```diff
@@ -5,86 +5,206 @@
...
+                 </div>
+                 <div class="tags">
+                    <h5>css</h5>
+                    <img src="img/Rectangle.png" alt="">
+                    <h5>html</h5>
+                    <img src="img/Rectangle.png" alt="">
+                    <h5>bootstrap</h5>
+                    <img src="img/Rectangle.png" alt="">
+                    <h5>Ruby</h5>
+                  </div>
```
Comment (Lucas-Masaba) [inline index.html:155]: - [ ] The list of technologies are all supposed to be wrapped in a `ul` tag with each individual technology having a `li` tag 👍

File: index.html [hunk 5]
```diff
@@ -5,86 +5,206 @@
...
```
Comment (Lucas-Masaba) [inline index.html:176]: - [ ] The list of technologies are all supposed to be wrapped in a `ul` tag with each individual technology having a `li` tag 👍

File: style.css [hunk 6]
```diff
@@ -255,3 +265,255 @@ ul li {
...
+
+.my-story button {
+  position: static;
+  width: 89px;
+  height: 48px;
+  left: 0;
+  top: 421px;
+  background: #7d73fc;
+  color: white;
+}
```
Comment (Lucas-Masaba) [inline style.css:389]: - [ ] Your button has a border around it and in the figma it is not present
### Your implementation
image
### Figma
image
You can try adding `border: none;` to this block 😎

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Josphat205  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Please add a background image as seen in Figma. You should also add an image to each project instead of living a white box as it is a requirement 👍 
### Your implementation
image
### Figma
image

- [ ] The text for your technologies is darker than that in the Figma. Please, make it lighter 
### Figma
image

### Your implementation
image

**Comment by Josphat205** [comment]
- Added images to my project
- Removed borders on my buttons
- Add ul to intended tags
- Added a professional Readme file

File: index.html [hunk 7]
```diff
@@ -5,86 +5,222 @@
...
+                        <li>html</li>
+                        <li> <img src="img/Rectangle.png" alt=""></li>
+                        <li>bootstrap</li>
+                        <li> <img src="img/Rectangle.png" alt=""></li>
+                        <li>Ruby</li>
+                      </ul>
+                     <div class="btn">
+                         <button type="button">See Project</button>
+                     </div>
+                </div>
```
Comment (MeqdamAlqudah) [inline index.html:168]: - [ ] As the last reviewer said please add a background image to each one of the cards as in Figma design
Your design 
image
Figma design 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @lomantech , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Josphat205** [comment]
- Added background images to match Figma design

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Josphat205,

CompleteDerekMullerGIF

You did great in fixing all the changes asked by the previous reviewers. Keep doing great 💪💪💪 
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**